HW1/task3.py

```diff
@@ -0,0 +1,26 @@
+from random import randint
+
+LOWER_LIMIT = 0
+UPPER_LIMIT = 1000
+MAX_ATTEMPTS = 10
+
+def play_game():
+    # Генерация случайного числа
+    num = randint(LOWER_LIMIT, UPPER_LIMIT)
+
+    print("Угадайте число от 0 до 1000. У вас есть 10 попыток.")
+
+    for attempt in range(1, MAX_ATTEMPTS + 1):
+        guess = int(input("Попытка {}: Ваше предположение: ".format(attempt)))
+
+        if guess == num:
+            print("Поздравляем! Вы угадали число {} за {} попыток.".format(num, attempt))
+            return
```

Comment:
Наверное проще прекратить цикл, чем использовать пустой `return`
```suggestion
            break
```